evalml/tests/component_tests/test_stacked_ensemble_regressor.py

```diff
@@ -125,15 +126,15 @@ def test_stacked_fit_predict_regression(X_y_regression, stackable_regressors):
     clf.fit(X, y)
     y_pred = clf.predict(X)
     assert len(y_pred) == len(y)
-    assert isinstance(y_pred, pd.Series)
-    assert not np.isnan(y_pred).all()
+    assert isinstance(y_pred, ww.DataColumn)
+    assert not np.isnan(y_pred.to_series()).all()
```

Comment:
I guess this looks like something we should add to our woodwork wishlist.  Shouldn't have to convert this to find out whether it is all Nan.  I forget which, whether numpy or pandas, but one of them has .isnan() defined.

Reply:
Indeed! I think both have some variant: numpy has `isnan` (https://numpy.org/doc/stable/reference/generated/numpy.isnan.html), pandas has `isna` (or also `isnull`?):
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.isna.html

Hoping the new WW API change will help with this though 😀